### PR TITLE
Fix #452 - 2.1.1 Update WdG

### LINKED ISSUE #452: 2.1.1 Update WdG

### PR DESCRIPTION
**Pull request by rzmoz**

### CODE REVIEW

File: warriorsOfTheDarkGods.cat [hunk 1]
```diff
@@ -4164,7 +4164,7 @@ Only a single unit may exit the same Gateway Marker in each Player Turn.</descri
       <rules>
         <rule id="c1a5-f7d9-42cd-7f9c" name="Favour of Cibaresh, God of Lust" hidden="false">
           <description>Universal Rule
-The model gains Feigned Flight and Strider. Units with more than half of their models with this Favour may declare Flee as a Charge Reaction even if they have Fearless. If doing so, the subsequent Rally Test is subject to Minimised Roll.</description>
+            The model gains Feigned Flight and Strider. Units with more than half of their models with this Favour may declare Flee as a Charge Reaction even if they have Fearless. If doing so, the subsequent Rally Test is subject to Minimised Roll.</description>
```
Comment (DarkSkyT9A) [inline warriorsOfTheDarkGods.cat]: Why are there unnecessary blanks before this line?

File: warriorsOfTheDarkGods.cat [hunk 2]
```diff
@@ -4178,7 +4178,7 @@ The model gains Feigned Flight and Strider. Units with more than half of their m
       <rules>
         <rule id="a0ad-f4e3-461f-1dfd" name="Favour of Savar, God of Pride" hidden="false">
           <description>Universal Rule
-Discipline Tests taken by units with at least one model with this Favour are subject to Minimised Roll.</description>
+            Discipline Tests taken by units with at least one model with this Favour are subject to Minimised Roll.</description>
```
Comment (DarkSkyT9A) [inline warriorsOfTheDarkGods.cat]: Why are there unnecessary blanks before this line?

File: warriorsOfTheDarkGods.cat [hunk 3]
```diff
@@ -4192,7 +4192,7 @@ Discipline Tests taken by units with at least one model with this Favour are sub
       <rules>
         <rule id="9e5a-ba7e-83d3-45a9" name="Favour of Nukuja, Goddess of Sloth" hidden="false">
           <description>Universal Rule
-Models with this Favour gain +1 Resilience. If a model with this Favour Declares a Charge against an enemy unit that is more than 10″ away or perform a March Move of more than 10″, this effect is lost until the start of the Melee Phase in the next Player Turn.</description>
+            Models with this Favour gain +1 Resilience. If a model with this Favour Declares a Charge against an enemy unit that is more than 10″ away or perform a March Move of more than 10″, this effect is lost until the start of the Melee Phase in the next Player Turn.</description>
```
Comment (DarkSkyT9A) [inline warriorsOfTheDarkGods.cat]: Why are there unnecessary blanks before this line?

File: warriorsOfTheDarkGods.cat [hunk 4]
```diff
@@ -4206,7 +4206,7 @@ Models with this Favour gain +1 Resilience. If a model with this Favour Declares
       <rules>
         <rule id="e227-4787-adce-c84e" name="Favour of Vanadra, Goddess of Wrath" hidden="false">
           <description>Attack Attribute - Melee
-The model part gains Lightning Reflexes and +1 Agility. Close Combat Attacks allocated towards the model gain +1 to hit. These effects are only applied in the first Round of each Combat.</description>
+            The model part gains Lightning Reflexes and +1 Agility. Close Combat Attacks allocated towards the model gain +1 to hit. These effects are only applied in the first Round of each Combat.</description>
```
Comment (DarkSkyT9A) [inline warriorsOfTheDarkGods.cat]: Why are there unnecessary blanks before this line?

File: warriorsOfTheDarkGods.cat [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<catalogue id="9cd6-2c84-c59e-7dd1" name="Warriors of the Dark Gods 2.2.4" revision="47" battleScribeVersion="2.02" authorName="Mikel2311, DarkSky" library="false" gameSystemId="aa64-1e8e-66fc-9abf" gameSystemRevision="2" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
+<catalogue id="9cd6-2c84-c59e-7dd1" name="Warriors of the Dark Gods 2.1.1" revision="48" battleScribeVersion="2.02" authorName="Mikel2311, DarkSky" library="false" gameSystemId="aa64-1e8e-66fc-9abf" gameSystemRevision="2" xmlns="http://www.battlescribe.net/schema/catalogueSchema">
```
Comment (DarkSkyT9A) [inline warriorsOfTheDarkGods.cat]: According to the file name, this is version 2.3 (seems to be an error in the document itself, as 2.1.1 is actually much older)

Comment (rzmoz) [reply warriorsOfTheDarkGods.cat]: I noticed that too during the changes but have now brought it up with the RT for clarification. 

Whatever they decide though, I believe that the BS files should correspond directly with the PDF they're trying to convey. The current WdG PDF says 2.1.1 and the previous was 2.2.x so the version decrease is originating in the source documents and not a typo from my side.

I suggest that the PDF and BS data files are kept in sync regardless. If this is a mistake, then the RT will release new PDF with a new version and we'll update the BS data file based on that.

File: warriorsOfTheDarkGods.cat (hunk 4 above)
Comment (rzmoz) [reply warriorsOfTheDarkGods.cat]: I'm updating the files manually and the lack of space makes it more difficult for me to collapse sections I don't need. It doesn't affect the printed output in BS (I checked).

**Pull request closed without merging**